src/03_modules.py

```diff
@@ -10,22 +10,42 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+# print('\n',dir(sys.argv))
+
+for line in dir(sys.argv):
```

Comment:
adding in the dir() wrapper here prints out more than just the sys arguments.
please remove it and push up the changes